src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/SwitchRules.groovy

```diff
@@ -5,12 +5,12 @@ import org.openkilda.model.FlowEncapsulationType
 import org.openkilda.model.FlowMeter
 import org.openkilda.model.SwitchId
 import org.openkilda.model.cookie.Cookie
+import org.openkilda.model.cookie.CookieBase.CookieType
 import org.openkilda.northbound.dto.v1.flows.PathDiscrepancyDto
 import org.openkilda.northbound.dto.v2.haflows.HaFlow
 import org.openkilda.testing.service.database.Database
 import org.openkilda.testing.service.northbound.NorthboundService
 
-
 class SwitchRules {
```

Comment:
the same for these changes: 
This part has been added from https://github.com/telstra/open-kilda/pull/5481/files and will not be a part of this PR after merging PR 5481 to develop. And this part was verified in the scope of HA-Flow server42 PR(can be skipped during review).